base/android/java/src/org/chromium/base/shared_preferences/BraveStrictPreferenceKeyChecker.java

```diff
@@ -6,24 +6,20 @@
 package org.chromium.base.shared_preferences;
 
 import org.chromium.base.BravePreferenceKeys;
-import org.chromium.base.Log;
 
 public class BraveStrictPreferenceKeyChecker extends StrictPreferenceKeyChecker {
-    private static String TAG = "BraveReflectionUtil";
-
     BraveStrictPreferenceKeyChecker(PreferenceKeyRegistry registry) {
         super(registry);
     }
 
     @Override
     public void checkIsKeyInUse(String key) {
         if (!isKeyInUse(key) && !BravePreferenceKeys.isBraveKeyInUse(key)) {
-            Log.e(
-                    TAG,
-                    "Key "
-                            + key
-                            + " is not registered for Brave's use. Either add it to"
-                            + " BravePreferenceKeys or remove it if it's not used.");
+            // Do nothing here for now. We might reconsider it in the future
```

Comment:
@SergeyZhukovsky what do you think of extending pref keys for specific feature in specific classes ? instead of including everything here in this file.

Reply:
yeah it could be done, we will need to refactor it in that case and move everything out to the corresponding classes.